state_machine/applications/system-check/main.py

```diff
@@ -1,53 +1,19 @@
-"""
-Python system check script for PyCubed satellite board
-PyCubed Mini mainboard-v02 for Pocketqube Mission
-* Author(s): Yashika Batra
-"""
-# print acknowledgement that test has started
-
 import tests
 import tests.i2c_scan
 import tests.sd_test
-# import tests.logging_infrastructure_test
 import tests.imu_test
-# import tests.radio_test
 import tests.sun_sensor_test
 import tests.coil_test
 import tests.burnwire_test
 import supervisor
 import tasko
 from print_utils import bold, normal, red, green
 
+# prevent board from reloading in the middle of the test
 supervisor.disable_autoreload()
 
 # initialize hardware_dict and result_dict
-result_dict = {
-    "LoggingInfrastructure_Test": ("", None),
-    "Basic_SDCard_Test": ("", None),
-    "IMU_AccGravity": ("", None),
-    "IMU_GyroStationary": ("", None),
-    "IMU_GyroRotating": ("", None),
-    "IMU_MagMagnet": ("", None),
-    "IMU_Temp": ("", None),
-    "Radio_ReceiveBeacon": ("", None),
-    "Radio_SendBeacon": ("", None),
-    "Sun-Y_Dark": ("", None),
-    "Sun-Y_Light": ("", None),
-    "Sun-Z_Dark": ("", None),
-    "Sun-Z_Light": ("", None),
-    "Sun-X_Dark": ("", None),
-    "Sun-X_Light": ("", None),
-    "Sun+Y_Dark": ("", None),
-    "Sun+Y_Light": ("", None),
-    "Sun+Z_Dark": ("", None),
-    "Sun+Z_Light": ("", None),
-    "Sun+X_Dark": ("", None),
-    "Sun+X_Light": ("", None),
-    "CoilDriverX": ("", None),
-    "CoilDriverY": ("", None),
-    "CoilDriverZ": ("", None),
-    "Burnwire": ("", None),
-}
+result_dict = {}
```

Comment:
Are there changes that need to be made to the other tests to handle this modification of `result_dict`?

Reply:
Nope

Reply:
It was used to print a list of tests that haven't been run, which we no longer do.